libarchive/archive_entry_private.h

```diff
@@ -48,6 +48,15 @@ struct ae_sparse {
 	int64_t	 length;
 };
 
+struct ae_mset_digest {
```

Comment:
Please use a bitmap for this rather than spending a full byte for each format.

Reply:
done.